File: Core/src/main/java/it/andrearossi/mcitaliaapi/exceptions/DefaultExceptionHandler.java
package it.andrearossi.mcitaliaapi.exceptions;

import it.andrearossi.mcitaliaapi.utils.DoubleRunnable;

public class DefaultExceptionHandler extends APIExceptionHandler {
	@Override
	public DoubleRunnable<ErrorCase, Exception> runnableAdapter() {
		return (errorCase, exception) -> exception.printStackTrace();
	}
}

File: src/main/java/it/andrearossi/mcitaliaapi/exceptions/DefaultExceptionHandler.kt
package it.andrearossi.mcitaliaapi.exceptions

import it.andrearossi.mcitaliaapi.utils.DoubleRunnable

class DefaultExceptionHandler : APIExceptionHandler() {
    override fun runnableAdapter(): DoubleRunnable<ErrorCase, Exception> {
        return DoubleRunnable { errorCase: ErrorCase?, exception: Exception -> exception.printStackTrace() }
    }
}